bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -1002,5 +1002,5 @@ spec:
   maturity: beta
   provider:
     name: Red Hat
-  replaces: rhpam-kogito-operator.7.13.3-1
-  version: 7.13.3-2
+  replaces: rhpam-kogito-operator.7.13.3-2
```

Comment:
I think the replaces here should be `rhpam-kogito-operator.7.13.3-1`. I think we only released one version of the Kogito bundle to the Catalog. Could you please double-check @spolti ?